apis/v1alpha1/instrumentation_types.go

```diff
@@ -70,6 +70,19 @@ type PythonSpec struct {
 	Image string `json:"image,omitempty"`
 }
 
+// Resource defines the configuration for the resource attributes, as defined by the OpenTelemetry specification.
+// See also: https://github.com/open-telemetry/opentelemetry-specification/blob/v1.8.0/specification/overview.md#resources
+type Resource struct {
+	// Attributes defines attributes that are added to the resource.
+	// For example environment: dev
+	// +optional
+	Attributes map[string]string `json:"resourceAttributes,omitempty"`
+
+	// K8sUIDAttributes defines whether K8s UID attributes should be collected (e.g. k8s.deployment.uid).
+	// +optional
+	K8sUIDAttributes bool `json:"k8sUIDAttributes,omitempty"`
```

Comment:
As this is a boolean, shouldn't the name perhaps be done in a way that aligns with it? Like, `[Enable|Add|Inject|Derive|Include]KubernetesAttributes`, or something like that?

Reply:
I like `Add..`